### DISCUSSION TITLE
rendering a table without a selected row?

### DISCUSSION THREAD

**Discussion by hellt** [charmbracelet/bubbletea#789] [Q&A]
Hey all,
Hopefully I am not trying to use the wrong tool for the job when trying to use bubble's table in a non-TUI setting.

What I am after is leveraging rich lipgloss styling support to output nicely looking tables which do not require user's input.
So what I do is basically this:

```go
t := table.New(
	table.WithColumns(columns),
	table.WithRows(rows),
	table.WithHeight(7),
)

s := table.DefaultStyles()
t.SetStyles(s)

fmt.Println(t.View())
```

but the table is printed with first row "selected". Is there a way to say that I don't want any rows to be selected?

**ADD1**
Apologies, this is a wrong repo. But the solutions seems to be as follows:

```go
s := table.Styles{
			Header: lipgloss.NewStyle().Bold(true).Padding(0, 1),
			Cell:   lipgloss.NewStyle().Padding(0, 1),
		}
		t.SetStyles(s)
```

**Comment by meowgorithm** [comment]
Without getting too deep into this, what should really happen is that we move the rendering portion of the table to Lip Gloss.

**Comment by pidanou** [comment]
It would be nice though to be able unselect rows and to make the selected row `nil`. 
Right now, it isn't even possible to set the cursor to -1 due to the `clamp` function on the `SetCursor` method. So once something is selected, you just can't unselect it.
Also the issue with the style trick is that a row is still selected under the hood, it is just not shown. It can induce errors for the users in some cases.

A mix of the list and the current tables would be great: a table where you can select/unselect/multi-select rows.
